cl_billing_pricing.module

```diff
@@ -74,6 +130,20 @@ function cl_billing_pricing_field_instance_settings_form($field, $instance) {
       '#title' => t('Valid License Types'),
     );
   }
+  
+  // We want to allow filtering out some of the licensable products.
+  if(!empty($settings['license_types'])) {
+    $license_types = ($instance['entity_type'] == 'commerce_license') ? array($instance['bundle']) : $settings['license_types'];
+    $form['licensable_products'] = array(
+      '#type' => 'select',
+      '#multiple' => TRUE,
+      '#options' => _cl_billing_pricing_get_product_options($license_types),
+      '#title' => t('Licensable products'),
+      '#description' => t('Include following products in Plans.'),
```

Comment:
Let's clarify this to indicate that if you pick nothing, all plans open to this license type (or the user's license types) will be included.